### PR TITLE
Settings 'docker' environment for running the WebAPI in Docker

### PR DESCRIPTION
**Pull request by AKlaus**

### CODE REVIEW

File: devops/run.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ if [ -z "$APP_SETTINGS" ]; then
     echo "APP_SETTINGS is required."
     exit 1
 else
-    echo "$APP_SETTINGS" > appsettings.json
+    echo "$APP_SETTINGS" > appsettings.docker.json
 fi
 
-dotnet WebApi.dll
\ No newline at end of file
+dotnet WebApi.dll --environment=docker
```
Comment (gregolsky) [inline devops/run.sh:10]: What does this one do anyway? Does it change they way the app works or is it just a marker?

Comment (AKlaus) [reply devops/run.sh:10]: Yes, it sets the current "environment" ([link to the docs](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/environments#environments)) that in our case is makes a difference only in picking `appsettings.docker.json` file (here, "_docker_" is the name of the environment) in addition to `appsettings.json`.

An alternative way of setting the environment would be adding `set ASPNETCORE_ENVIRONMENT=docker` to the `Dockerfile`

Comment (gregolsky) [reply devops/run.sh:10]: ok gotcha. Now I recall what it used to and will update my PR.

Comment (gregolsky) [reply devops/run.sh:10]: Actually, I'll leave it as is - we can't deployment ready to go settings.json into the container. It's hosted as public image on DockerHub under `ravendbsamples/yabt`. We would have exposed the DB credentials this way. I'll set the DB credentials in ECS directly.

**Comment by AKlaus** [comment]
Merged #13 instead. Closing this one

**Pull request closed without merging**